lang/en/docs/software-directory/machine-learning/python-ml/components.md

```diff
@@ -86,7 +89,17 @@ when [ the predict workflow is generated](../../../properties-directory/non-scal
 ### Subworkflow: Machine Learning
 
 This subworkflow is where a user's requested machine learning units reside. This subworkflow is generally the one that
-users are expected to modify, to add or remove different machine learning workflow units.
+users are expected to modify, to add or remove different machine learning workflow units. In the above diagram,
+we find the following units:
+
+| Order | Unit Name                     | Flavor                                       |
+|-------|-------------------------------|----------------------------------------------|
+| 1     |`Setup Packages and Variables` | `pyml:setup_variables_packages`              |
+| 2     | ` Data Input`                 | `pyml:data_input:read_csv:pandas`            |
+| 3     | `Data Standardize`            | `pyml:pre_processing:standardization:sklearn`|
+| 4     | `Model Train and Predict`     | `pyml:model:multilayer_perceptron:sklearn`   |
+| 5     | `Parity Plot`                 | `pyml:post_processing_parity_plot_matplotlib`|
```

Comment:
Shouldn't this be `pyml:post_processing:parity_plot:matplotlib`??

Reply:
Ah, that's a typo in the name of the flavor in the web-app. I'll correct it here and ticket out the fix